### ISSUE TITLE
How disable eslint-plugin-react(react-in-jsx-scope) rules?

### ISSUE THREAD

**Issue by FabianMontoya** [oxc-project/oxc#2061]
I'm using the OXC VS Code extension and I'm getting the `slint-plugin-react(react-in-jsx-scope): 'React' must be in scope when using JSX help: When using JSX, `<a />` expands to `React.createElement("a")`. Therefore the `React` variable must be in scope.` warning from this extension, but in my `.eslintrc.json` I have:

```js
"react/react-in-jsx-scope": "off",
"react/jsx-props-no-spreading": "off",
```

I want to remove that intrusive warning from the editor without import React, that isn't needed nowadays.

---

![Screenshot 2024-01-16 at 12 21 02](https://github.com/oxc-project/oxc/assets/37662924/47f16fe2-d7f5-4f3d-ab25-ef8f3739cbc6)

![Screenshot 2024-01-16 at 12 21 44](https://github.com/oxc-project/oxc/assets/37662924/eceb1640-90c4-4d2f-90b0-6ef41aa79875)

**Comment by Boshen** [comment]
This doesn't sound like a correctness rule, I'll remove it from the defaults.

Reading configuration from vscode is still work in progress.

**Comment by Boshen** [comment]
Is this from v0.2.1? Previous versions had a bug turning on these unwanted rules.

**Comment by FabianMontoya** [comment]
I'll check the version

**Comment by FabianMontoya** [comment]
I updated and reload all my VS Code and it's solved, thanks!

![Screenshot 2024-01-16 at 22 03 53](https://github.com/oxc-project/oxc/assets/37662924/f654a77c-1991-4eb6-a7f6-7c2cc555156b)

**Comment by Boshen** [comment]
My mistake releasing a badly tested version 😅
